### ISSUE TITLE
Dropdown offset is too large on Edge

### ISSUE THREAD

**Issue by cuberoot** [adobe/spectrum-web-components#647]
### Expected Behaviour

<img width="212" alt="Screen Shot 2020-04-24 at 3 19 59 PM" src="https://user-images.githubusercontent.com/154097/80171674-15801a80-863f-11ea-802f-01ec99086bd4.png">

### Actual Behaviour

<img width="238" alt="Screen Shot 2020-04-24 at 3 39 06 PM" src="https://user-images.githubusercontent.com/154097/80172695-c7b8e180-8641-11ea-8ecc-43fb00e4ca93.png">

#### Steps to Reproduce

Open https://opensource.adobe.com/spectrum-web-components/components/dropdown in pre-chromium Edge

Or, do a local build of the docs and paste this code into the [`<head>` tag of the docs](https://github.com/adobe/spectrum-web-components/blob/master/documentation/index.html#L3)

```jsx
        <script type="text/javascript">
            window.ShadyDOM = { force: true }
        </script>
```

You can then see the problem in Chrome as well.

#### Platform and Version

Pre-chromium Edge, or any browser that needs web components polyfills

**Comment by Westbrook** [comment]
```
#contents ::slotted(*) {
    position: relative;
}
```
In `active-overlay.css` _seems_ to address this, but I'm not 100% sure that makes sense and/or is OK. Thoughts?

**Comment by cuberoot** [comment]
That makes sense to me. It should establish the active-overlay as the parent coordinate system, which would have been my understanding of how it should be. Thank you for all of your help with these Edge-cases :-)

**Comment by Westbrook** [comment]
@cuberoot can you confirm whether the addition outlined above (and included in #646) has corrected this?

**Comment by Westbrook** [comment]
Considering this corrected. Please feel free to reopen if this issue persists.
